packages/app/src/app/pages/common/UserMenu/index.tsx

```diff
@@ -52,48 +52,48 @@ export const UserMenu: FunctionComponent & {
 
         <Menu.List>
           <Menu.Link to={profileUrl(user.username)}>
-            <Stack align="center" gap={1}>
-              <Icon name="profile" size={24} />
-              <Text>My Profile</Text>
+            <Stack align="center" gap={2}>
+              <Icon name="profile" size={16} />
+              <Text>Profile</Text>
             </Stack>
           </Menu.Link>
 
           <Menu.Divider />
 
           <Menu.Link to={dashboardUrl()}>
-            <Stack align="center" gap={1}>
-              <Icon name="dashboard" size={24} />
+            <Stack align="center" gap={2}>
+              <Icon name="dashboard" size={16} />
```

Comment:
Should the fill be transparent? Right now, the icon background is black and does not blend with the menu background.

![dgwcw8-3000 preview csb app_dashboard_settings](https://user-images.githubusercontent.com/24959348/190205060-f7c931e7-5ca7-4c07-b18d-6f3cddba3705.png)